custodia/cli/__init__.py

```diff
@@ -176,7 +201,49 @@ def handle_plugins(args):
     'plugins', help='List plugins')
 parser_plugins.set_defaults(
     func=handle_plugins,
-    command='plugins')
+    command='plugins',
+    sub='plugins',
+    name=None,
+)
+
+
+def error_message(args, exc):
+    out = StringIO()
+    parts = urlparse(args.server)
+
+    if args.debug:
+        traceback.print_exc(file=out)
+        out.write('\n')
+
+    out.write("ERROR: Custodia command '{args.sub} {args.name}' failed.\n")
+    if args.verbose:
+        out.write("Custodia server '{args.server}'.\n")
+
+    if isinstance(exc, RequestsHTTPError):
```

Comment:
it isn't a blocker, it was just a suggestion, but you're right, we can refactor later.